utils/protobuf-schema-utilities/src/main/java/io/apicurio/registry/utils/protobuf/schema/FileDescriptorUtils.java

```diff
@@ -374,6 +386,36 @@ private static EnumDescriptorProto enumElementToProto(EnumType enumElem) {
         return builder.build();
     }
 
+    private static DescriptorProtos.ServiceDescriptorProto serviceElementToProto(Service serviceElem) {
+        ServiceDescriptorProto.Builder builder = ServiceDescriptorProto.newBuilder().setName(serviceElem.name());
+
+        for (Rpc rpc : serviceElem.rpcs()) {
+            MethodDescriptorProto.Builder methodBuilder = MethodDescriptorProto
+                    .newBuilder()
+                    .setName(rpc.getName())
+                    .setInputType(getTypeName(rpc.getRequestType().toString()))
+                    .setOutputType(getTypeName(rpc.getResponseType().toString()));
```

Comment:
We need to support streaming option as well.

Reply:
Updated